app/views/step_by_step_pages/index.html.erb

```diff
@@ -107,6 +107,7 @@
   </div>
 
   <div class="govuk-grid-column-three-quarters">
+    <p class="govuk-body govuk-!-margin-bottom-1"><%=pluralize(@step_by_step_pages.count, "step by step")%></p>
```

Comment:
Looks like we're missing spaces after the `%=` and before the `%>` here.

Reply:
Update and fixed up.